### DISCUSSION TITLE
Dose table widget has $pollingInterval working same as in chart widgets and state widgets

### DISCUSSION THREAD

**Discussion by alkoumi** [filamentphp/filament#9118] [Help] [answered]
I have a dashboard with many types of widgets

State Widgets
Chart Widgets
Table Widgets

I use the ```protected static ?string $pollingInterval = '10s';``` to keep the user updated 
with the data changes without refreshing the page

State Widgets and Chart Widgets work well in every database change reflected in the widget
But Table Widgets does not get the changes

What is the problem?

I hope anyone can help ❤️

**Comment by VincvanAs** [comment, marked as answer]
You should use ->poll('10s') on the table

**Reply by alkoumi** [reply]
Many Many Thanks brother @VincvanAs it worked like a charm  
💝
